README.md

```diff
@@ -63,6 +63,31 @@ This approach has advantages to the one above:
 - The development code is the same as the build.
 - No need to edit the plugins text file.
 
+## Custom channel presentation proxy development
+
+Instead of bundle the custom channel presentation bundle inside the apps
+directory, it's possible to work with a proxy for development.
+
+1. Run the custom channel presentation development server without mock data:
+
+```
+$ cd kolibri-channel-custom-web-app/template-ui
+$ VUE_APP_USE_MOCK_DATA=false yarn serve
```

Comment:
You can also add this to a `.env.development.local` file. See https://cli.vuejs.org/guide/mode-and-env.html#environment-variables